packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ProjectInfo/icons.tsx

```diff
@@ -1,5 +1,34 @@
 import React from 'react';
 
+export const TwitterIcon = props => (
```

Comment:
No need for passing `props`

```suggestion
export const TwitterIcon = () => (
```

Reply:
For now they are not used, we still pass them to make sure that if it's ever needed they are there ready to be reused because eventually they will all be passed into the components folder